spec/OpenLineage.yml

```diff
@@ -68,6 +68,7 @@ components:
         runId:
           description: The id of the run, unique relative to the job
           type: string
+          format: uuid
```

Comment:
Do we want runIds to be UUIDs?

Reply:
updated the PR so runid in the parent facet is the same as this runid